app/src/main/java/com/findmyrecycling/ProfileScreen.kt

```diff
@@ -29,9 +29,9 @@ class ProfileScreen : ComponentActivity() {
 }
 @Composable
 fun ProfileOptions(name: String) {
-    var profile by remember{ mutableStateOf("")}
-    var mySavedLocations by remember{ mutableStateOf("")}
-    var mySavedSearches by remember{ mutableStateOf("")}
+    var profile by remember{ mutableStateOf("") }
+    var mySavedLocations by remember{ mutableStateOf("") }
+    var mySavedSearches by remember{ mutableStateOf("") }
```

Comment:
Added spacing before curly braces (here and other places as well), as per Kotlin convention.
https://kotlinlang.org/docs/coding-conventions.html#lambdas